src/uct/ib/ud/base/ud_iface.c

```diff
@@ -584,8 +584,7 @@ ucs_status_t uct_ud_iface_query(uct_ud_iface_t *iface, uct_iface_attr_t *iface_a
     iface_attr->cap.am.max_zcopy       = iface->super.config.seg_size - sizeof(uct_ud_neth_t);
     iface_attr->cap.am.align_mtu       = uct_ib_mtu_value(uct_ib_iface_port_attr(&iface->super)->active_mtu);
     iface_attr->cap.am.opt_zcopy_align = UCS_SYS_PCI_MAX_PAYLOAD;
-    iface_attr->cap.am.max_hdr         = uct_ib_iface_hdr_size(iface->config.max_inline,
-                                                               sizeof(uct_ud_neth_t));
+    iface_attr->cap.am.max_hdr         = iface->super.config.seg_size - sizeof(uct_ud_neth_t);
```

Comment:
does it work for ud_mlx5?

Reply:
yes, RC reports non-zero inline

Reply:
i think it doesn't work. does gtest pass for ud_mlx5?